x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/page.tsx

```diff
@@ -32,51 +32,58 @@ import { DataFrameAnalysisConfigType } from '../../../../../common/types/data_fr
 export const Page: FC<{
   jobId: string;
   analysisType: DataFrameAnalysisConfigType;
-}> = ({ jobId, analysisType }) => (
-  <Fragment>
-    <NavigationMenu tabId="data_frame_analytics" />
-    <EuiPage data-test-subj="mlPageDataFrameAnalyticsExploration">
-      <EuiPageBody style={{ maxWidth: 'calc(100% - 0px)' }}>
-        <EuiPageContentHeader>
-          <EuiPageContentHeaderSection>
-            <EuiTitle>
-              <h1>
-                <FormattedMessage
-                  id="xpack.ml.dataframe.analytics.exploration.title"
-                  defaultMessage="Analytics exploration"
-                />
-                <span>&nbsp;</span>
-                <EuiBetaBadge
-                  label={i18n.translate(
-                    'xpack.ml.dataframe.analytics.exploration.experimentalBadgeLabel',
-                    {
-                      defaultMessage: 'Experimental',
-                    }
-                  )}
-                  tooltipContent={i18n.translate(
-                    'xpack.ml.dataframe.analytics.exploration.experimentalBadgeTooltipContent',
-                    {
-                      defaultMessage: `Data frame analytics are an experimental feature. We'd love to hear your feedback.`,
-                    }
-                  )}
-                />
-              </h1>
-            </EuiTitle>
-          </EuiPageContentHeaderSection>
-        </EuiPageContentHeader>
-        <EuiPageContentBody style={{ maxWidth: 'calc(100% - 0px)' }}>
-          <EuiSpacer size="l" />
-          {analysisType === ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION && (
-            <OutlierExploration jobId={jobId} />
-          )}
-          {analysisType === ANALYSIS_CONFIG_TYPE.REGRESSION && (
-            <RegressionExploration jobId={jobId} />
-          )}
-          {analysisType === ANALYSIS_CONFIG_TYPE.CLASSIFICATION && (
-            <ClassificationExploration jobId={jobId} />
-          )}
-        </EuiPageContentBody>
-      </EuiPageBody>
-    </EuiPage>
-  </Fragment>
-);
+  isTraining?: string | boolean;
+}> = ({ jobId, analysisType, isTraining }) => {
+  if (isTraining !== undefined) {
+    isTraining = isTraining === 'true';
+  }
```

Comment:
I think `isTraining` should always come here as a boolean

Reply:
Agreed - updated to always be a boolean 43a33e33877d0893f3f0041df1c7240c1df45860